resources/functionMap.php

```diff
@@ -72,10 +72,10 @@
 'AMQPChannel::__construct' => ['void', 'connection'=>'AMQPConnection'],
 'AMQPChannel::basicRecover' => ['void', 'requeue='=>'bool'],
 'AMQPChannel::commitTransaction' => ['void'],
-'AMQPChannel::getChannelId' => ['int'],
+'AMQPChannel::getChannelId' => ['int<1,65535>'],
```

Comment:
This is defined as a 16bit unsigned int in the spec, but 0 triggers an error always.